package.json

```diff
@@ -16,12 +18,17 @@
   },
   "main": "index.js",
   "devDependencies": {
+    "babel-core": "^6.24.1",
+    "babel-preset-env": "^1.4.0",
     "chai": "^3.5.0",
+    "eslint": "^3.19.0",
+    "eslint-plugin-json": "^1.2.0",
```

Comment:
It doesn't look like you are actually configuring `.eslintrc` to use `eslint-plugin-json`.  Do we need that plugin if we aren't using it?